activerecord/lib/active_record/persistence.rb

```diff
@@ -1017,9 +1017,10 @@ def touch(*names, time: nil)
       _raise_readonly_record_error if readonly?
 
       attribute_names = timestamp_attributes_for_update_in_model
-      attribute_names |= names.map! do |name|
+      attribute_names = (attribute_names | names).map! do |name|
         name = name.to_s
-        self.class.attribute_aliases[name] || name
+        name = self.class.attribute_aliases[name] || name
+        verify_readonly_attribute(name) || name
```

Comment:
I realize this line is same [as in `update_columns`](https://github.com/rails/rails/blob/a680a684d1c7e4080e994e4c4fc1fa38d1fe73b0/activerecord/lib/active_record/persistence.rb#L835), but it made me check if `verify_readonly_attribute` ever returns a value ([it doesn't](https://github.com/rails/rails/blob/a680a684d1c7e4080e994e4c4fc1fa38d1fe73b0/activerecord/lib/active_record/persistence.rb#L1133)).  Is there any objection to putting `verify_readonly_attribute(name)` on a separate line?